src/lib.rs

```diff
@@ -30,14 +30,17 @@ pub fn clone(lock: Lock) -> anyhow::Result<()> {
         );
     }
 
+    let mut clone_args = vec!["clone", "--branch", lock.branch.as_str()];
+    if shallow {
+        clone_args.push("--depth");
+        clone_args.push("1");
```

Comment:
I thought about an option to specify any depth, but I believe this will never be used. If we agree that we need to be able to specify any depth, I change.

Reply:
Hmm, should we make `shallow` the default?

Reply:
I'm all in for that. I believe `zkevm-llvm` is mostly used in CI, so there should not be any problems with making it the default one.

Reply:
@hedgar2017 , do you think it makes sense to rework the option to `--full`, instead of `--shallow`? As I can see StructOpt does not provide a way to have default for `bool`.

Reply:
@antonbaliasnikov yeah, flipping the option is ok. Maybe `--deep` sounds better :)

Reply:
Done